docs/can-view-import.md

```diff
@@ -29,6 +29,52 @@ Dynamically import *MODULE_NAME*; the scope within the template is a [Promise](h
 
 @param {moduleName} [MODULE_NAME] A module that this template depends on.
 
+@signature `<can-dynamic-import from="MODULE_NAME" value:to="*MODULE_REF"/>`
+
+Dynamically import a module from within a [can-stache] template. Since there is no subtemplate to attach the [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) to as the current scope, you must export the Promise's resolved value to the template's refs scope using [can-view-import.value].
+
+```
+<can-dynamic-import from="components/tabs" value:to="*tabsWidget" />
+{{#if *tabsWidget}}
+	<tabs-widget />
+{{/if}}
+
+{{! other can-reflect-promise keys also work, as does the *REFERENCE shorthand from can-stache-bindings }}
+
+<can-dynamic-import from="components/tabs" 
+	isPending:to="*tabsWidgetPending"
+	isRejected:to="*tabsWidgetError"
+	*tabs-widget-promise />
```

Comment:
does `this:to` work?

Reply:
It does.  Thanks for pointing it out.  You can see some live results at https://ebony-port.glitch.me/